subprojects/gradle/build-checks/src/gradleTest/kotlin/com/avito/android/build_checks/ConfigurationCacheCompatibilityTest.kt

```diff
@@ -33,7 +39,9 @@ internal class ConfigurationCacheCompatibilityTest {
         return gradlew(
             projectDir,
             "help",
-            dryRun = true,
+            "-Pkapt.incremental.apt=true",
```

Comment:
Why do we need this? How it affects the build check?

Reply:
no more, when i extracted room check

Reply:
Sorry, still didn't get why we use this flag here.

Reply:
it was used with different set of tasks to check "configuration cache", i just forgot to remove it